fhir-server/liberty-config/config/default/fhir-server-config-db2.json

```diff
@@ -128,7 +128,7 @@
                     "trustAll": true
                 },
                 "cos" : { 
-                    "maxResourcesThreshold": 200,
+                    "maxResourcesThreshold": 200000,
```

Comment:
@prb112 Robin and I discussed this one and we think measuring this in "thousands of resources" was a little odd.  we think it should be safe to stay an integer because we shouldn't have more than 2 billion resources in a single export.
side-note:  I think there is another spot in our persistence layer with a similar limitation.